packages/block-library/src/page-list/edit.js

```diff
@@ -342,8 +342,7 @@ export default function PageListEdit( {
 					<PanelBody title={ __( 'Edit this menu' ) }>
 						<p>{ convertDescription }</p>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen, while you add page-list block and check the inspector panels, when `convertToLinks` option is true.

| Before | After     |
| :-------- | :------- |
| <img width="364" alt="page-list-2-before" src="https://github.com/user-attachments/assets/8c666af3-21db-43d4-9f15-a95d5113ccdb"> | <img width="364" alt="page-list-2-after" src="https://github.com/user-attachments/assets/2b8018ae-ac2a-4a25-9a10-273eff5efdfd"> |